integration_tests/e2e/calendar-overtime.cy.ts

```diff
@@ -18,13 +19,13 @@ context('A staff member can view their overtime calendar', () => {
     })
     cy.login()
 
-    Page.verifyOnPageTitle(CalendarPage, moment().format('MMMM YYYY'))
+    Page.verifyOnPageTitle(CalendarPage, format(Date.now(), 'MMMM yyyy'))
```

Comment:
annoyingly `date-fmt` has different conventions for date format - https://date-fns.org/v2.28.0/docs/format